Mlem/App/Views/Pages/Instance/InstanceUptimeView.swift

```diff
@@ -18,15 +17,40 @@ struct InstanceUptimeView: View {
     @State var showingAllDowntimes: Bool = false
     
     let instance: any Instance
-    let uptimeData: UptimeData
+    @State var uptimeData: UptimeData
+    
+    var uptimeRefreshTimer = Timer.publish(every: 30, tolerance: 0.5, on: .main, in: .common)
+        .autoconnect()
     
-    @ViewBuilder
     var body: some View {
+        ScrollView {
+            content
+        }
+        .background(.themedGroupedBackground)
+        .onReceive(uptimeRefreshTimer) { _ in
+            Task {
+                let uptimeStatus = await loadUptimeData(instance: instance)
+                switch uptimeStatus {
+                case .success(let uptimeData):
+                    withAnimation(.easeOut(duration: 0.2)) {
+                        self.uptimeData = uptimeData
+                    }
+                case .unavailable:
+                    assertionFailure("Uptime data unavailable.")
+                case .failure(let error):
+                    handleError(error)
+                }
+            }
+        }
+    }
+    
+    @ViewBuilder
+    var content: some View {
         VStack(alignment: .leading, spacing: 0) {
             section { summary }
                 .padding(.bottom, 10)
```

Comment:
IMO we need more top padding
```suggestion
                .padding(.bottom, 10)
                .padding(.top, 16)
```